target/linux/ramips/image/mt7620.mk

```diff
@@ -869,6 +892,29 @@ define Device/tplink_archer-c20i
 endef
 TARGET_DEVICES += tplink_archer-c20i
 
+define Device/tplink_re200-v1
+  MTK_SOC := mt7620a
+  DEVICE_VENDOR := TP-Link
+  TPLINK_BOARD_ID := RE200-V1
+  TPLINK_HWID := 0x02000001
+  TPLINK_HWREV := 0x00000001
+  KERNEL := $(KERNEL_DTB)
+  KERNEL_INITRAMFS := $(KERNEL_DTB) | tplink-v1-header -e -O
+  IMAGES+= factory.bin
+  IMAGE/sysupgrade.bin := mktplinkfw sysupgrade -e -O
+  IMAGE/factory.bin := mktplinkfw factory -e -O -v 3.14.2
+  TPLINK_HEADER_VERSION := 1
+  IMAGE_SIZE := 7936k
+  TPLINK_FLASHLAYOUT := 8Mlzma
+  DEVICE_PACKAGES := kmod-mt76x0e
+  DEVICE_MODEL := RE200
+  KERNEL_ENTRY := 0x8000c310
+  KERNEL_LOADADDR := 0x80000000
+  DEVICE_VARIANT := v1
+  SUPPORTED_DEVICES += re200
```

Comment:
Drop this line.

Reply:
You mean the `SUPPORTED_DEVICES +=` line? Tbh, I have no clue what this does, I just copied it from the other devices.

Reply:
I know ...